src/SFA.DAS.AssessorService.Application.Api/Controllers/ExternalApi/ExternalApiDataSyncController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AssessorService.Application.Api.Middleware;
+using SFA.DAS.AssessorService.Application.Api.Properties.Attributes;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using SFA.DAS.AssessorService.Application.Handlers.ExternalApi.DataSync;
+
+namespace SFA.DAS.AssessorService.Application.Api.Controllers.ExternalApi
+{
+    [Authorize(Roles = "AssessorServiceInternalAPI")]
+    [Route("api/v1/externalapidatasync")]
```

Comment:
Filename = ExternalApiDataSyncController.cs
Classname =ExternalApiDataSyncController
Route = "api/v1/externalapidatasync"
Method = RebuildExternalApiSandbox

externally this would be api/v1/externalapidatasync/rebuild-sandbox" according to your HttpPost attribute